distributed/pom.xml

```diff
@@ -37,7 +37,7 @@
     <name>OrientDB Distributed Server</name>
 
     <properties>
-        <hz.version>3.10.6</hz.version>
+        <hz.version>3.11.7</hz.version>
```

Comment:
I think we can update also up to 3.12.x, we may have some additional feature there tu use as well

Reply:
When I wanted to choose a version, I wanted to find a version that has Kubernetes support and doesn't require changes on our side. I tried out also 3.12.x, but that requires some changes in our code. I think not that much, but still we need to change something. I forgot exactly what.
So, if you think 3.12 has something we need, I can try it again, and also see what needs to change.